frontend/ui/widget/buttonprogresswidget.lua

```diff
@@ -47,18 +48,24 @@ function ButtonProgressWidget:update()
     local button_width = math.floor(self.width / self.num_buttons) - 2*button_padding - 2*button_margin - 2*button_bordersize
     for i = 1, self.num_buttons do
         local highlighted = i <= self.position
+        local is_default = i == self.default_position
         local margin = button_margin
         if self.thin_grey_style and highlighted then
             margin = 0 -- moved outside button so it's not inverted
         end
+        local extra_border_size = 0
+        if not self.thin_grey_style and is_default then
+            -- make the border a bit bigger on the default button
+            extra_border_size = Size.border.thin
+        end
         local button = Button:new{
             text = "",
             radius = 0,
             margin = margin,
             padding = button_padding,
-            bordersize = button_bordersize,
+            bordersize = button_bordersize + extra_border_size,
             enabled = true,
-            width = button_width,
+            width = button_width - 2*extra_border_size,
```

Comment:
No concern about height?

Reply:
No. The height can overflow (see pic of the black style). For the width, we don't want that to keep having even dstributed squares all allong the progress bar, so we care about having the final width (inside + borders) similar.
That's for the black style we don't use :) so no worry at all for the thin grey style.

(and uh, uhnit what ? :)